apps/test/unit/p5lab/reducersTest.js

```diff
@@ -1,12 +1,22 @@
+// Require statements can change behavior based on order.
+// We are keeping the original order of the require statements.
```

Comment:
Is there a specific reason for this file why we want to preserve the original order?

Reply:
I'll work on removing it in a quick follow up PR